pinda-fun/pinda-fun
# Add create new room screen

### Changes
Static screen for new room page.
- Styled according to mockup
- Added qr code generator
- Added telegram and whatsapp sharing links
- Link page up with landing page and vice versa

#### Desktop Screenshot
<img width="1280" alt="Screenshot 2019-10-16 at 4 11 38 PM" src="https://user-images.githubusercontent.com/25261058/66900579-aabaaa80-f02f-11e9-9f77-d398e62e7ba6.png">

#### Mobile Screenshot
<img width="213" alt="Screenshot 2019-10-16 at 4 10 34 PM" src="https://user-images.githubusercontent.com/25261058/66900580-ab534100-f02f-11e9-9820-85f09298a7a3.png">


### How to test
Remember to `yarn install`, then `yarn start` and click the **Start New Game** button to go to this screen. Clicking **Cancel** should bring you back to landing.
That's about it :)
